src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -731,6 +732,14 @@ public class SimpleEvents {
 				)
 				.since("INSERT VERSION");
 
+		Skript.registerEvent("Player Exp Cooldown Change", SimpleEvent.class, PlayerExpCooldownChangeEvent.class, "[player] (exp cooldown change)")
```

Comment:
This registration should be behind a class/method exists check, like your ExprExpCooldown is.

Reply:
Disregard, you changed this as I was reviewing lol